release-notes/unreleased/LUN-272-incorrect-names-for-additional-areas.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: LUN-272
+show_in_stores: true
+platforms:
+  - android
+  - ios
+de: Fehlerhalfte Titel von individuellen Bereichen behoben
```

Comment:
Was this on production? Otherwise I'd set `show_in_stores: false`

Reply:
yes it was.

Reply:
You sure? axios-cache-interceptor was only added in februrary, but at least the version from the play store is from january:
![image](https://user-images.githubusercontent.com/29095487/159471792-144bc30f-7bf6-409e-bc74-a49381eff633.png)
![image](https://user-images.githubusercontent.com/29095487/159471903-8b0e0af3-d15c-41ce-83c1-2c233567e300.png)

Reply:
hmh, you are right :thinking: 
Thought Julia found this on live app. Will remove.